sympy/logic/boolalg.py

```diff
@@ -1915,6 +2093,213 @@ def POSform(variables, minterms, dontcares=None):
     return And(*[_convert_to_varsPOS(x, variables) for x in essential])
 
 
+def ANFform(variables, truthvalues):
```

Comment:
good choice of names, but would be even more good if you document them. As to what they are?

Reply:
`variables` are just variables, for example: `a, b, c,..` etc.
`truthvalues` are values of the last column of an expression's truth table.
I have no idea how to document them

Reply:
Follow something like this:
```
A short description of method.

Parameters
----------
param1 : object-type
    what does this parameter mean/do?

param2 : object-type
    what does this parameter mean/do?

param3 : object-type, is this optional if yes then write "optional" (mention the default)
    and possible attainable values of param3 and what do they mean?

Returns
----------
return1: type_of_object:
    what does this return mean?

Examples
------------
here goes examples
```

It is not necessary that every method be documented, but it will help understand the methods better.

Reply:
done